cmd/globalAlignmentAnchor/globalAlignmentAnchor.go

```diff
@@ -25,40 +25,78 @@ func writeToFileHandle(file io.Writer, species1 bed.Bed, species2 bed.Bed, score
 	exception.PanicOnErr(err)
 }
 
+// helper function: check if maf entry passes checks
+func matchMafPass(assembly_species1 string, assembly_species2 string, chrom_species1 string, chrom_species2 string, species1_SrcSize int, species2_SrcSize int, species1_ChromStart int, species1_ChromEnd int, species2_ChromStart int, species2_ChromEnd int) bool {
+	pass := true
+
+	// chrom should be the same between species1 and species2 (special case: panTro6 chr2A, chr2B both count as a match to hg38 chr2)
+	pT6_hg38_chr2 := ((assembly_species1 == "hg38" && assembly_species2 == "panTro6" && chrom_species1 == "chr2" && ((chrom_species2 == "chr2A") || (chrom_species2 == "chr2B"))) || (assembly_species1 == "panTro6" && assembly_species2 == "hg38" && ((chrom_species1 == "chr2A") || (chrom_species1 == "chr2B")) && chrom_species2 == "chr2")) // the special case written as a bool variable
```

Comment:
Looks like you're hard coding in the assemblies. Have you thought about this being user-specified to help this be more generalized?

Reply:
That's a good idea! But this is the best way I can think of now to address chimp chromosome 2A and 2B matching human chromosome 2, and I believe I can keep this hard-coded because no matter what species the user is trying to align, if they have hg38 and panTro6, they will need this adjustment for the chr2

Reply:
I don't think we need it for now, but we could try to think of something more general, like we could consider "chrX" to match "chrXY", as long as X was a number (i.e. one or more digit) and Y was a letter.